pkg/cloud/packet/client.go

```diff
@@ -180,6 +180,26 @@ func (p *Client) NewDevice(ctx context.Context, req CreateDeviceRequest) (*metal
 
 	serverCreateOpts := metal.CreateDeviceRequest{}
 
+	var sshKeyInput []metal.SSHKeyInput
+	if len(packetMachineSpec.SSHKeys) > 0 {
+		for _, key := range packetMachineSpec.SSHKeys {
+			sshKeyInput = append(sshKeyInput, metal.SSHKeyInput{
+				Label: metal.PtrString(fmt.Sprintf("cluster-api-provider-packet-%s", req.MachineScope.Name())),
```

Comment:
All SSHKeys will have the same label per machine.  Should we use a tag for the repeated values and a unique value per label? I'm pretty sure labels are optional. If we can't find a meaningful unique label, we could omit them. The tag would tell us what we need to know about the SSHKey.  If Label is required, perhaps it could suffix the label you've provided with some tail bytes of the SSH Key? (not the comment section of the key).